server/pulp/plugins/util/publish_step.py

```diff
@@ -577,9 +602,11 @@ def get_distributor_type(self):
 
     def publish(self):
         """
+        DEPRECATED
```

Comment:
Only because it's historically been difficult to prevent those warnings from showing up in front of users and customers, who can't do anything about them, but definitely get concerned.